sdk/pom.xml

```diff
@@ -830,5 +836,11 @@
       <version>0.5.160304</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
```

Comment:
I'm not sure if auto-value has been discussed before. We should check with @davorbonaci on whether or not we want to introduce this dependency.